README.md

```diff
@@ -34,15 +38,15 @@ More detailed instructions are on our ["Usage"](/docs/USAGE.md) page.
 # Run the app using Docker
 ### Setup
 1. Download and install [Docker](https://docs.docker.com/get-started/)
-1. Pull the [latest Docker image for this project](https://github.com/orgs/MobilityData/packages/container/package/gtfs-validator). For example, `docker pull ghcr.io/mobilitydata/gtfs-validator:v2.0.0`.
+1. Pull the [latest Docker image for this project](https://github.com/orgs/MobilityData/packages/container/package/gtfs-validator). For example, `docker pull ghcr.io/mobilitydata/gtfs-validator:fix-debian-version-used-in-docker-image`.
```

Comment:
After this text changes I'm not sure it will be obvious what the user should run to get the latest image. I'd suggest providing more specific instructions that include copying the value show in the text box at the top of the https://github.com/orgs/MobilityData/packages/container/package/gtfs-validator page.

Reply:
Actually, if we can just use `snapshot` instead here that would be greatly preferred (see below).

Reply:
At present, Docker images are only published at reslease or manually when needed (as we did for the [latest one](https://github.com/orgs/MobilityData/packages/container/gtfs-validator/2374496?tag=fix-debian-version-used-in-docker-image)) - Do you think that we should publish a new docker image for each snapshot version of the validator?

Reply:
>Do you think that we should publish a new docker image for each snapshot version of the validator?

Yes, I think that would be helpful if someone wants to pull in a bug fix before the release of the next version (i.e., in the case of fixing the debian version). We have the gtfs-realtime-validator set up to publish a new docker image on each commit to main branch.

Reply:
> Actually, if we can just use snapshot instead here that would be greatly preferred (see below).

Would following the format: snapshot-sha float our boat in this case?

Reply:
Ideally you want something that's not going to change frequently so you don't need to keep updating the docs on every commit to the main branch.

See https://github.com/CUTR-at-USF/gtfs-realtime-validator#docker for example.

We publish the RT validator to Docker Hub and every commit gets the `latest` TAG, and Docker Hub always returns that when you do `docker pull cutrusf/gtfs-realtime-validator`.

Reply:
Please tell me if https://github.com/MobilityData/gtfs-validator/pull/905/commits/e776d1b751488a1048eae3c65f13da83fe42db4f clarifies this :)

Reply:
Much better, thanks!